.github/workflows/dockerhub-mirror.yml

```diff
@@ -0,0 +1,51 @@
+# taken from https://github.com/rblaine95/dockerhub-mirror/
+
+name: Mirror Dockerhub
+env:
+  REGISTRY: ghcr.io
+  CONTAINER_REPO: ${{ github.repository }}
+  GO_VERSION: 1.23.4
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Missing top level permission, otherwise we will get warning: https://github.com/inspektor-gadget/inspektor-gadget/pull/3765